data_structures/src/lib.rs

```diff
@@ -1,6 +1,7 @@
 //! data structures
 
-#![deny(rust_2018_idioms)]
+// Removed due to flatbuffers generated code
```

Comment:
`FIXME`

Reply:
Instead of removing, it'll be better to turn off this by applying `#[allow(rust_2018_idioms)]` to generated mod as there may be other hand-writed code in the future.

Reply:
We tried to apply `#[allow(rust_2018_idioms)]` after declaring the `#![deny(rust_2018_idioms)]` but it seems that outer attributes cannot override inner attributes. Any ideas?